package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "prepublish": "npm test",
     "clean": "rm -rf dist && mkdir dist",
-    "cp-ts": "cp lib/razorpay.d.ts dist/",
+    "cp-types":"mkdir dist/types && cp lib/types/* dist/types && cp lib/utils/*d.ts dist/utils",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Can we generate a single ts file and ship that? Do we have to copy all the type files here? Is that the standard practice? 

If you see https://github.com/razorpay/blade/blob/master/packages/blade/package.json#L37, there seems to be a single ts file for all the components.

Reply:
@anuraghazra Any comments on the above? Should we have a single type file or is shipping multiple files fine?